docs/UPGRADING.md

```diff
@@ -4,6 +4,53 @@ Here you can find a list of migration guides to handle breaking changes between
 
 ## 0.22.0
 
+### package `github.com/arduino/arduino-cli/httpclient` has been moved to `github.com/arduino/arduino-cli/arduino/httpclient`
```

Comment:
- `github.com/arduino/arduino-cli/httpclient.UserAgent` was moved to `github.com/arduino/arduino-cli/configuration.UserAgent` 
- `github.com/arduino/arduino-cli/httpclient.NetworkProxy` was moved to `github.com/arduino/arduino-cli/configuration.NetworkProxy`